Tools/autotest/sim_vehicle.py

```diff
@@ -239,6 +239,8 @@ def find_root_dir():
 group_build.add_option("-b", "--build-target", default=None, type='string', help="override SITL build target")
 group_build.add_option("-s", "--build-system", default="waf", type='choice', choices=["make", "waf"], help="build system to use")
 group_build.add_option("", "--no-rebuild-on-failure", dest="rebuild_on_failure", action='store_false', default=True, help="if build fails, do not clean and rebuild")
+group_build.add_option("", "--waf-configure-arg", action="append", dest="waf_configure_args", type="string", default=[], help="extra arguments to pass to waf in its configure step")
+group_build.add_option("", "--waf-build-arg", action="append", dest="waf_build_args", type="string", default=[], help="extra arguments to pass to waf in its build step")
```

Comment:
@guludo I'll see if I can neaten them next time I've got an excuse to code-churn there.  Possibly just breaking "help" onto the next line will be sufficient.

I tend to maximise my window when I work on chunks of code like that, so it doesn't really bother me.